backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Sales_Invoice_Details_local_currency_conversion.sql

```diff
@@ -0,0 +1,119 @@
+ */
+
+CREATE OR REPLACE FUNCTION de_metas_endcustomer_fresh_reports.Docs_Sales_Invoice_Details_local_currency_conversion(IN p_C_invoice_id numeric)
+    RETURNS TABLE
+            (
+                c_invoice_id                numeric(10),
+                IsPrintLocalTaxReporting    boolean,
+                local_iso_code              bpchar(3),
+                conversion_rate             numeric,
+                conversion_date             timestamp without time zone,
+                local_taxbaseamt            numeric,
+                local_taxamt                numeric
+            )
+AS
+$$
+
+
+SELECT it.C_Invoice_ID,
+       CASE
+           WHEN i.m_inout_id IS NULL OR i.c_order_id IS NULL   THEN FALSE
+           WHEN i.isprintlocalcurrencyinfo = 'N'               THEN FALSE
+           WHEN bp.isprintlocalcurrencyinfo = 'N'
+               AND   i.isprintlocalcurrencyinfo <> 'Y'        THEN FALSE
+                                                               ELSE TRUE
+       END             AS IsPrintLocalTaxReporting,
+       cu.iso_code AS local_iso_code,
+       currencyrate(c.c_currency_id,
+                    cu.c_currency_id,
+                    getlocaltaxreportingconversionratedate(tr.taxreportingratebase, tr.c_calendar_id, i.dateinvoiced, m.movementdate),
+                    tr.c_conversiontype_id,
+                    i.ad_client_id,
+                    i.ad_org_id) AS conversion_rate,
+       CASE
+           WHEN i.m_inout_id IS NULL THEN NULL
+                                     ELSE
+                                         getlocaltaxreportingconversionratedate(
+                                                 tr.taxreportingratebase,
+                                                 tr.c_calendar_id,
+                                                 i.dateinvoiced,
+                                                 m.movementdate)
+       END AS conversion_date,
+       CASE
+           WHEN i.m_inout_id IS NULL THEN NULL
+                                     ELSE
+                                         currencyconvert(SUM(TaxBaseAmt),
+                                                         c.c_currency_id,
+                                                         cu.c_currency_id,
+                                                         i.dateinvoiced,
+                                                         tr.c_conversiontype_id,
+                                                         i.ad_client_id,
+                                                         i.ad_org_id)
+       END AS local_taxbaseamt,
+       CASE
+           WHEN i.m_inout_id IS NULL THEN NULL
+                                     ELSE
+                                         currencyconvert(SUM(TaxAmt),
+                                                         c.c_currency_id,
+                                                         cu.c_currency_id,
+                                                         i.dateinvoiced,
```

Comment:
why aren't we use conversion_date here?

Reply:
forget to change it back after testing something